react-locate-contractor/src/Pages/Login.tsx

```diff
@@ -16,7 +16,6 @@ const Login = ({ config }: any) => {
         return <div>Loading ... </div>;
     }
 
-
     return authState.isAuthenticated ?
```

Comment:
Usually if there is a file that differs by only spacing, I would revert it back so it doesn't show up as changed